src/components/Select/index.tsx

```diff
@@ -33,15 +36,23 @@ export const Select = ({
       <components.Placeholder {...props}>{placeholder}</components.Placeholder>
     )
   }
-
+  const labelCSS = label && { marginBottom: "8px" }
   return (
-    <ReactSelect
-      {...props}
-      options={options}
-      styles={SelectStyled}
-      components={{ Placeholder, DropdownIndicator }}
-      isMulti={isMulti}
-      isClearable={isClearable}
-    />
+    <Box>
+      <Box css={{ ...labelCSS }}>
+        <Box as="label" htmlFor={name} css={{ color: "$primary500" }}>
```

Comment:
O correto é atribuir o `id` ao `htmlFor`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for